README.md

```diff
@@ -1,5 +1,9 @@
+[![Build Status](https://travis-ci.org/PenneyGadget/lookingfor.svg?branch=master)](https://travis-ci.org/PenneyGadget/lookingfor)
```

Comment:
Here are the "getting started" instructions: https://travis-ci.org/getting_started

Did you hit "sync" to pull in your repos? After that you go to https://travis-ci.org/profile/<your username>, find the LookingFor repo, and flick the repo switch to "on". Does that make sense and line up with what you see?